lib/oxidized/input/ssh.rb

```diff
@@ -24,7 +24,11 @@ def connect node
       @log = File.open(Oxidized::Config::Log + "-#{@node.ip}-ssh", 'w') if Oxidized.config.input.debug?
       port = vars(:ssh_port) || 22
       if proxy_host = vars(:proxy)
-        proxy =  Net::SSH::Proxy::Command.new("ssh #{proxy_host} nc %h %p")
+        if proxy_sshio = vars(:ssh_tunnel)
+          proxy =  Net::SSH::Proxy::Command.new("ssh #{proxy_host} -W %h:%p")
```

Comment:
I know is some :nail_care: but what do you think of not duplication the whole line?

``` ruby
options = if proxy_sshio = vars(:ssh_tunnel)
            "-W %h:%p"
          else
            "nc %h %p"
          end

proxy =  Net::SSH::Proxy::Command.new("ssh #{proxy_host} #{options}")
```